tuf/metadata/v1/tuf_test.go

```diff
@@ -0,0 +1,274 @@
+/*
+Copyright 2018 Google Inc. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1
+
+import (
+	"errors"
+	"reflect"
+	"testing"
+
+	"github.com/GoogleCloudPlatform/runtimes-common/tuf/metadata"
+	"github.com/GoogleCloudPlatform/runtimes-common/tuf/testutil"
+	"github.com/GoogleCloudPlatform/runtimes-common/tuf/types"
+)
+
+func TestRootMetadata(t *testing.T) {
+	tuf := TUF{testutil.TestTUFConfig,
```

Comment:
nit: use the key: value initializer form.

Reply:
hmm what is that? i googled but did not find an example.

Reply:
done